internal/connections/bundledconfig/docker/fleet/heartbeat.yaml.tmpl

```
@@ -0,0 +1,63 @@
+  receivers:
+    heartbeat:
+      interval: 10s
+      environment: docker
+      auth_check:
+        url: ${env:OBSERVE_COLLECTOR_URL}
+        headers:
+          authorization: ${env:OBSERVE_AUTHORIZATION_HEADER}
+  processors:
+    k8sattributes/heartbeat:
+      extract:
+        otel_annotations: true
+        metadata:
+          - k8s.namespace.name
+          - k8s.node.name
+          - k8s.pod.name
+          - k8s.container.name
+          - service.name
+        labels:
+          # Extract app.kubernetes.io/* labels from the pod as the full tag.
+          - tag_name: $1
+            key_regex: (app\.kubernetes\.io/.+)
+            from: pod
+      passthrough: false
+      pod_association:
+      - sources:
+        - from: resource_attribute
+          name: k8s.pod.ip
+      - sources:
+        - from: resource_attribute
+          name: k8s.pod.uid
+      - sources:
+        - from: connection
+    resource/heartbeat:
+      attributes:
```

Should this also add `observe.agent.instance.id` for use elsewhere, ie:
```suggestion
      attributes:
        - key: observe.agent.instance.id
          action: upsert
          value: ${env:OBSERVE_AGENT_INSTANCE_ID}
```